src/org/zaproxy/zap/authentication/AuthenticationMethodType.java

```diff
@@ -149,22 +151,23 @@ public abstract AuthenticationMethod loadMethodFromSession(Session session, int
 	 * @param authMethod the auth method to persist
 	 * @throws UnsupportedAuthenticationMethodException the unsupported authentication method
```

Comment:
The UnsupportedAuthenticationMethodException is an unchecked exception, it's not required to be declared.